### PR TITLE
New parts models added.

### PR DESCRIPTION
**Pull request by andrzej-jankowski**
New models:
- Part
- PartModel

This is a prelude to the new parts management...

### CODE REVIEW

File: src/ralph_assets/models_parts.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        on_delete=models.PROTECT,
+    )
+    part_environment = models.ForeignKey(
+        DeviceEnvironment,
+        default=None,
+        null=True,
+        on_delete=models.PROTECT,
+    )
+    warehouse = models.ForeignKey(Warehouse, on_delete=models.PROTECT)
+    created = models.DateTimeField(
```
Comment (mkurek) [inline src/ralph_assets/models_parts.py]: why don't you use `TimeTrackable` from `lck.django` (even if we are going to gradually eliminate lck.django from ralph, we'll probably have to develop our own version of this model, used in many places across whole ralph, so one more place in my opinion will not make much more work to do during replacing it)?

Comment (andrzej-jankowski) [inline src/ralph_assets/models_parts.py]: ``TimeTrackable`` from ``lck.django`` it's not only ``created`` and ``modified`` fields. It's also ``cache_version`` and I think that we don't need it in our DB...

Comment (andrzej-jankowski) [inline src/ralph_assets/models_parts.py]: OK, we can remove it later...

Comment (mkurek) [inline src/ralph_assets/models_parts.py]: ok - i've looked at `TimeTrackable` and `auto_now_add` and `auto_now` do pretty much the same - so never mind and forget about my notice :wink:

**Comment by ar4s** [comment]
:OK:

**Comment by xliiv** [comment]
:ok:

**Pull request merged**